### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by mrprotocoll**
### The above updates were made

- Setup new  rails api project
- Added linters
- setup greetings endpoint
- Update README
- Enable CORS

### CODE REVIEW

**Review by zainsadaqat — COMMENTED** [review]
# ❌ ❌ ❌Invalid Code Review Request ❌ ❌ ❌
# Hello @mrprotocoll  👋  

## Something went wrong 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for this reason:

- ### You have submitted a project, where the Front-End repository is completely empty as you can see in the Screenshot.

Screenshot 2023-06-08 at 12 39 10 AM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :greetings
+  end
+  # Defines the root path route ("/")
+  # root "articles#index"
```
Comment (Ismailco) [inline config/routes.rb:7]: - [ ] [OPTIONAL] Kindly make sure to create a root router

**Review by Ismailco — APPROVED** [review]
### Hi @mrprotocoll, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Setup Rails correctly
✔️ Used Postgresql & seed database
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (mrprotocoll) [reply config/routes.rb:7]: Thank you @Ismailco , i'll surely make that changes

**Pull request merged**